### DISCUSSION TITLE
Changing Mount Information

### DISCUSSION THREAD

**Discussion by JYasin** [juicedata/juicefs#2797] [General]
i want to change the mount information

JuiceFS:myjfs               1.0P  384K  1.0P   1% /root/test/juicefs-test
JuiceFS:juicefs             1.0P  384K  1.0P   1% /root/test/juicefs-test
JuiceFS:myjfs               1.0P  384K  1.0P   1% /root/test/juicefs-test
JuiceFS:myjfs               1.0P  372K  1.0P   1% /root/test/jfs-2
JuiceFS:myjfs               1.0P  376K  1.0P   1% /root/test/jfs-3
JuiceFS:juicefs             1.0P  392K  1.0P   1% /root/test/jfs-4

how i can rename the **JuiceFS** in **JuiceFS:myjfs**   to everyone?

**Comment by davies** [comment]
It's hard coded, but you can overwrite that by `-o fsname=XXXX`, which depends on `libfuse`
